docker/images/pinot/etc/jmx_prometheus_javaagent/configs/broker.yml

```diff
@@ -1,71 +1,71 @@
 rules:
-- pattern: "\"org.apache.pinot.common.metrics\"<type=\"BrokerMetrics\", name=\"pinot.broker.(\\w+).authorization\"><>(\\w+)"
+- pattern: "\"org.apache.pinot.common.metrics\"<type=\"BrokerMetrics\", name=\"pinot.broker.(.*?).authorization\"><>(\\w+)"
   name: "pinot_broker_authorization_$2"
   cache: true
   labels:
     table: "$1"
-- pattern: "\"org.apache.pinot.common.metrics\"<type=\"BrokerMetrics\", name=\"pinot.broker.(\\w+)\\.documentsScanned\"><>(\\w+)"
+- pattern: "\"org.apache.pinot.common.metrics\"<type=\"BrokerMetrics\", name=\"pinot.broker.(.*?)\\.documentsScanned\"><>(\\w+)"
```

Comment:
(not introduced in this PR)
Some of the pattern escapes `.` as `\\.`, some not. Do we need to escape it?

Reply:
Great question. I think the correct pattern is having `\\.` but because `.` is a wildcard, it works regardless.

Ex:
pinotAbroker would still match regex `pinot.broker` 

Requires another PR to fix this pattern.

Reply:
Agree we can fix it in a separate PR